client/src/components/existing-request/existing-request-display.vue

```diff
@@ -478,16 +478,17 @@ export default class ExistingRequestDisplay extends Mixins(NrAffiliationMixin, C
           await newReqModule.patchNameRequestsByAction(action)
           break
       }
-    } else {
-      newReqModule.setActiveComponent('InvalidActionMessage')
```

Comment:
This `else` caught errors from confirmAction() and displayed a message that didn't make sense (and additionally an error modal). So I removed it.
Now, only the error modal displays, and the user remains on this page, which feels right to me.